project/dexter-metrics/src/test/com/samsung/sec/dexter/metrics/util/MetricUtilTest.java

```diff
@@ -28,5 +35,80 @@ public void test() {
 		
 	//MetricUtil.getFunctionLOCArray(source, 0, 14);
 	}
-
+	
+	@Test
+	public void MetricUtilTest_NonexistantFile() {
+		MetricUtil tester = new MetricUtil();
+		int start = 0;
+		int end = 10;
+		DexterRuntimeException exception = null;
+		try {
+			tester.getFunctionLOCArray(".\\ThereIsNoSuchFile.nope", start, end);
```

Comment:
getFunctionLOCArray() is static method. It should be called like 'MetricUtil.getFunctionLOCArray()'.
And it would be nice to put the function 'Assert.fail()' between 46 and 47 line to verify that the exception occurrs.